header/dependent-css/header-apps.css

```diff
@@ -0,0 +1,238 @@
+@import "esri-global-shared";
+
+.esri-header-apps p {
+	line-height: 17px;
+}
+
+.esri-header-apps .hide {
+	display: none;
+}
+
+.esri-header-apps a {
+	color: #333333;
+	text-decoration: none;
+}
+
+.esri-header-apps .appSwitcher-title {
+	font-size: 20px;
+}
+
+/* Apps Container */
+
+.esri-header-apps-control {
+	cursor: pointer;
+	padding: 16px 12px 8px 10px;
+}
+
+.esri-header-apps-control:hover {
+	box-shadow: inset 0 -3px 0 0 rgba(0, 121, 193, .5);
+}
+
+.esri-header-apps-control:active, .esri-header-apps-control[aria-expanded="true"] {
+	box-shadow: inset 0 -3px 0 0 #0079c1;
+}
+
+/* Calcite Styles */
+
+.esri-header-apps .block {
```

Comment:
@gschnall I think this layout could be done without the weight of the entire block group code. 

I would use flex box properties on this maybe.

Reply:
@paulcpederson good call! I'm not sure if the styling is too great but we'll be implementing a new design for this soon anyway.